packages/spright-components/src/chat/input/tests/chat-input.spec.ts

```diff
@@ -268,4 +268,25 @@ describe('ChatInput', () => {
             expect(page.getSendButtonTextContent()).toEqual('Send it!');
         });
     });
+
+    describe('tabindex', () => {
```

Comment:
Our other tests on tabindex are covering three cases (default, set to value, set and then clear value): https://github.com/ni/nimble/blob/55a4ce35d7064b9bae809825b4f72aaacab05fdf/packages/nimble-components/src/button/tests/button.spec.ts#L15-L51